libsolidity/lsp/LanguageServer.cpp

```diff
@@ -128,6 +128,76 @@ Json::Value semanticTokensLegend()
 	return legend;
 }
 
+struct MarkdownBuilder
+{
+	std::stringstream result;
+
+	MarkdownBuilder& code(std::string const& _code)
+	{
+		auto constexpr SolidityLanguageId = "solidity";
+		result << "```" << SolidityLanguageId << '\n' << _code << "\n```\n\n";
+		return *this;
+	}
+
+	MarkdownBuilder& paragraph(std::string const& _text)
+	{
+		if (_text.empty())
```

Comment:
```suggestion
		if (!_text.empty())
```
?

Reply:
@nikola-matic i've fixed that and also moved the handler logic into its own handler file to keep LanguageServer.{cpp,h} small.